rollup.config.js

```diff
@@ -36,72 +44,104 @@ const postcssPlugins = [
   postcssReporter(),
 ];
 
-export default [
+// This list includes common plugins shared between each output format.
+// The list is sorted alphabetically.
+const basePlugins = [
+  // Transpile sources using our custom babel preset.
+  babel({
+    exclude: ['node_modules/**'],
+    runtimeHelpers: true,
+    ...babelOptions(),
+  }),
+  // For shimming nodejs builtins
+  builtins(),
+  // To remove comments, trim trailing spaces, compact empty lines,
+  // and normalize line endings
+  cleanup(),
+  // To convert CJS modules to ES6
+  commonjs({
+    include: 'node_modules/**',
+    // Explicitly specify "unresolvable" named exports
+    // https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports
+    namedExports: {
+      'node_modules/react/index.js': [
+        'Children',
+        'Component',
+        'createElement',
+        'isValidElement',
+      ],
+      'node_modules/react-is/index.js': ['isValidElementType'],
+      'node_modules/flatpickr/dist/l10n/de.js': ['German'],
+    },
+  }),
+  // To convert JSON files to ES6
+  json(),
+  // To automatically externalize `dependencies` and `peerDependencies`
+  // so that they do not end up in the bundle.
+  // See also https://medium.com/@kelin2025/so-you-wanna-use-es6-modules-714f48b3a953
+  peerDepsExternal({
```

Comment:
peer deps external should go first. [ref](https://www.npmjs.com/package/rollup-plugin-peer-deps-external)